runtime/cmake/platform/toolcfg/openxl.cmake

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# Copyright IBM Corp. and others 2024
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+################################################################################
+
+list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
+	-O3
+	-fstack-protector
+)
+
+list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -fno-rtti)
+
+if(NOT OMR_OS_ZOS)
+	list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qsuppress=1540-1087:1540-1088:1540-1090)
```

Comment:
I don't think Open XL supports any options beginning with `-q`; this looks like stuff carried over from xlc.

Reply:
Done, removed the unused options, and also rebased with latest commit.